### PR TITLE
Added information on CNG support in mage.exe

### PR DESCRIPTION
**Pull request by rpetrusha**
## Added information on CNG support in mage.exe

Internal review link: [Mage.exe (Manifet Generation and Editing Tool](https://review.docs.microsoft.com/en-us/dotnet/framework/tools/mage-exe-manifest-generation-and-editing-tool?branch=pr-en-us-5943)

//cc @Tanya-Solyanik

### CODE REVIEW

File: docs/framework/tools/mage-exe-manifest-generation-and-editing-tool.md [hunk 1]
```diff
@@ -36,14 +36,15 @@ Mage [commands] [commandOptions]
   
 <a name="NewUpdate"></a>   
 ## New and Update Command Options  
- The following table shows the options supported by the `-New` and `-Update` commands.  
+
+The following table shows the options supported by the `-New` and `-Update` commands.
```
Comment (mairaw) [inline docs/framework/tools/mage-exe-manifest-generation-and-editing-tool.md]: nit: end with :

File: docs/framework/tools/mage-exe-manifest-generation-and-editing-tool.md [hunk 2]
```diff
@@ -36,14 +36,15 @@ Mage [commands] [commandOptions]
...
 |**-appm, -AppManifest** `manifestPath`||Deployment manifests.|Inserts a reference to a deployment's application manifest into its deployment manifest.<br /><br /> The file indicated by `manifestPath` must exist, or Mage.exe will issue an error. If the file referenced by `manifestPath` is not an application manifest, Mage.exe will issue an error.|  
-|**-cf, -CertFile** `filePath`||All file types.|Specifies the location of an X509 digital certificate for signing a manifest. This option can be used in conjunction with the **-Password** option, if the certificate requires a password.|  
+|**-cf, -CertFile** `filePath`||All file types.|Specifies the location of an X509 digital certificate for signing a manifest. This option can be used in conjunction with the **-Password** option if the certificate requires a password.<br/><br/>Starting with the .Net Framework 4.6.2 SDK, which is distributed with Visual Studio and with Windows SDK, mage.exe signs manifests with  CNG as well as CAPI certificates.|
```
Comment (mairaw) [inline docs/framework/tools/mage-exe-manifest-generation-and-editing-tool.md]: .Net -> .NET 
italicize mage.exe
remove extra spaces between with and CNG
Can't you download the SDK separately as well?

**Review by mairaw — APPROVED** [review]
Approving the text but with some comments left to be addressed.

**Review by Tanya-Solyanik — COMMENTED** [review]
Looks good

**Pull request merged**